components/eventDetails/EventDetails.tsx

```diff
@@ -124,6 +125,16 @@ const EventDetails = ({ event }: EventDetailsProps) => {
             </View>
           )}
       </View>
+      <MapSnippet
+        style={{ width: "100%", height: "30%" }}
```

Comment:
I wonder if we want the height to be a percentage or a fixed number. If possible, find someone with a different screen size to test it and see if it looks good. @suryatiru If you're available, check it out on your device.